scripts/system_performance/controller/method_mapping.py

```diff
@@ -0,0 +1,218 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Handles User input (N instruments, N methods and N additional method arguments), mapping
+user specified methods formatted as strings to their corresponding method in controller applying
+any additional arguments specified.
+
+:returns a dictionary of dictionaries containing statistics per instrument, per method
+"""
+
+# Dependencies
+import logging
+
+from scripts.system_performance.controller.statistics_computation import QueryHandler
+from scripts.system_performance.models import query_argument_constructor
+
+
+class MethodSelectorConfigurator(object):
+    """Class containing logic to call N methods specified by user for N instruments + any additional
+     method arguments specified"""
+
+    def __init__(self):
+        # create_method_mappings
+        pass
+
+    @staticmethod
+    def create_method_mappings():
```

Comment:
could define in __init__ as a member variable? means we don't have to create the dict each time we want to use in instead it is initialised with the class